debian/postinst

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+# Copyright 2015, EMC, Inc.
+
+set -e
+
+# Load debconf to ask use input parameters
+. /usr/share/debconf/confmodule
+
+SERVICES=("on-http" "on-taskgraph" "on-dhcp-proxy" "on-tftp" "on-syslog", "rackhd")
+
+#############################################
+# Check if you have 2 NIC as below:
+#  eth0 for the public network - providing access to RackHD APIs, and providing routed (layer3) access to out of band network for machines under management
+#  eth1 for dhcp/pxe to boot/configure the machines. and IP configurated as static ( 172.31.128.0/22 )
+# Parameter #1 ($1) is the NIC name of the control port for RacKHD
+#############################################
+check_NIC(){
```

Comment:
To be strictly, the network config, isc-dhcp-server config (event http/tftp static files downloading) is not the responsibility for RackHD debian package. It belongs to the deployment guide. Not every customer use isc-dhcp-server, not every customer prefers such kind of network config.